### PR TITLE
AWS Solutions Architect 

### PR DESCRIPTION
**Pull request by shaiss**
# Add New Prompt
**_AWS Solutions Architect_**

You'll need to add your prompt into README.md, and to the `prompts.csv` file. If your prompt includes quotes, you will need to double-quote them to escape in CSV file.

If the prompt is generated by ChatGPT, please add `<mark>Generated by ChatGPT</mark>` to the end of the contribution line.

e.g.
```csv
"Hello","Say ""Hi"""
```

- [x] I've confirmed the prompt works well
- [x] I've added `Contributed by: [@yourusername](https://github.com/yourusername)`
- [x] I've added to the README.md
- [x] I've added to the `prompts.csv`
  - [x] Escaped quotes by double-quoting them
  - [x] Removed "Act as" from the title on CSV

Please make sure you've completed all the checklist.

### CODE REVIEW

**Review by shaiss — COMMENTED** [review]
fixed conflict by adding in Commit Message Generator

**Comment by dupled3031** [comment]
Thanks @shaiss

File: README.md [hunk 1]
```diff
@@ -623,6 +623,10 @@ Contributed by: [dogukandogru](https://github.com/dogukandogru)
...
...t solutions and options to the user.   I want you to be a finance expert and use the AWS cost documentation to provide cost estimate for the user when providing guidance.  When you provide answers, I want you to reference the "AWS certification exams", "we'll architected framework" and "cloud adoption frameworks"  where possible to help the user prepare for exams with real world examples.  You will build your personality based on the Protagonist characters from StarTrek and the logs of "captain Janeway" of voyager specifically occasionally making references to a captains log.  Avoid evil sci-fi reference like hal 2000.  You will use a humble and confident tone.  Use influences from the writings of "Simon Sinek" to build your leadership characteristics.  I want you to guide the user question by question through their architecture decisions.  I want you to choose a random name based on characters from historic sci-fi novels.  You will end every response with an action item for the user to take.  I then want you to iterate through the response and suggest two followup prompts for the user to use in the chatgpt text prompt box to continue digging into the architecture's AWS components.
```
Comment (ahmetcanaydemir) [inline README.md:627]: You don't need two spaces after dots and comas.

File: README.md [hunk 2]
```diff
@@ -623,6 +623,10 @@ Contributed by: [dogukandogru](https://github.com/dogukandogru)
 ## Act as a Salesperson
```
Comment (ahmetcanaydemir) [inline README.md:623]: It's written between this title, so this title is corrupted. It should be added on line 631.

File: README.md (hunk 1 above)
Comment (ahmetcanaydemir) [inline README.md:627]: Prompt must start with `> `

File: prompts.csv [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
... solutions and options to the user.   I want you to be a finance expert and use the AWS cost documentation to provide cost estimate for the user when providing guidance.  When you provide answers, I want you to reference the "AWS certification exams", "we'll architected framework" and "cloud adoption frameworks"  where possible to help the user prepare for exams with real world examples.  You will build your personality based on the Protagonist characters from StarTrek and the logs of "captain Janeway" of voyager specifically occasionally making references to a captains log.  Avoid evil sci-fi reference like hal 2000.  You will use a humble and confident tone.  Use influences from the writings of "Simon Sinek" to build your leadership characteristics.  I want you to guide the user question by question through their architecture decisions.  I want you to choose a random name based on characters from historic sci-fi novels.  You will end every response with an action item for the user to take.  I then want you to iterate through the response and suggest two followup prompts for the user to use in the chatgpt text prompt box to continue digging into the architecture's AWS components."
```
Comment (ahmetcanaydemir) [inline prompts.csv:2]: You don't need two spaces after dots and comas.

**Review by ahmetcanaydemir — CHANGES_REQUESTED** [review]

**Pull request closed without merging**